docs/releases/2.5.1.rst

```diff
@@ -0,0 +1,74 @@
+Translate Toolkit 2.5.1
+***********************
+
+*Released on 25 April 2020*
+
+This release contains improvements and bug fixes.
+
+
+Changes
+=======
+
+Formats and Converters
+----------------------
+
+- PO
+
+    - Avoid adding extra space on empty comment lines
+    - Several performance improvements
+
+- Android resources
+
+    - Workaround broken plural handling for languages missing other tag
+    - Fix setting rich content with comments
+
+- YAML
+
+    - Fixed handling of empty keys
+    - Improved round trip preserving of comments and style
+
+- TS
+
+    - Avoid duplicating plurals definition
+    - Fixed possible crash on adding new translations
+
+- INI
+
+    - Now supported on Python 3 thanks to iniparse support for it
+
+- Catkeys
+
+    - The catkeys format now has support for fingerprint calculation
+
+- JSON
+
+    - Allow usage of BOM in JSON files
+
+- MO
```

Comment:
MO is usually including in PO, as these are Gettext.

Reply:
These are really MO specific (bbebec2608886d4e7589a94317aa371560b091e0 and 5ef74e47e900123335c2504d528e6b26774183d5)

Reply:
I meant that we usually put MO related changes under PO, but stating they are MO related. I am Ok with this as is anyway.